NDTensors/src/abstractarray/fill.jl

```diff
@@ -1,16 +1,16 @@
-using .SetParameters: DefaultParameters, specify_parameters
-using .Unwrap: unwrap_type
+using .TypeParameterAccessors: specify_defaults
+using .Unwrap: unwrap_array_type
 
 function generic_randn(
   arraytype::Type{<:AbstractArray}, dim::Integer=0; rng=Random.default_rng()
 )
-  arraytype_specified = specify_parameters(unwrap_type(arraytype), DefaultParameters())
+  arraytype_specified = specify_defaults(unwrap_array_type(arraytype))
```

Comment:
Definitely nice to drop the `DefaultParameters()`-based interface.

I'm thinking maybe we should call this `specify_default_parameters` instead of just `specify_defaults`. `specify_defaults` could refer to all sorts of things (say, specifying default keyword arguments, or something else).

Reply:
I agree that name is better, I have update the code across the library to the name `specify_default_parameters`